.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,22 @@
+name: Python application
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install pynwb
+      run: |
+        pip install git+https://github.com/NeurodataWithoutBorders/pynwb.git@cd3c352087a159dfab635c0cde4239fbede0973a
```

Comment:
Does this line mean this file needs to update every time pynwb updates (or the validate_spec method of pynwb updates)? Perhaps the spec validator should live in this repo.

Reply:
I would like the spec validator to be importable, which means it would need to live in a repo that contains a setup.py and is released on pypi. For that reason I put it in pynwb, but it might be better suited for hdmf. I included a specific commit because the validator spec has not yet been merged into pynwb, so a simple pip install will not install it.